plugins/colored-man-pages/nroff

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# The whole point of this wrapper is to replace emboldening factor -u0 with
+# the provided $_NROFF_U (which is always 1) under certain circumstances on
+# Solaris.
```

Comment:
Change the comment to say `-u1` instead of `-u0`.

Reply:
Wait, what? But it's `-u0` which gets replaced with `-u1`! Or do you mean just get rid of mentioning `_NROFF_U`?

Reply:
Sorry, "replace `-u0` with `-u1`". Brain fart 😂

Reply:
Makes sense. [x2]